bin/git/update-hooks

```diff
@@ -19,11 +56,15 @@ install_hook_wrapper() {
         # If a hook already exists and is a file:
         if [ -f $HOOK_DIR/$hook ]; then
             step_name "Moving existing '$hook' script to '$hook-local'"
-            mv $HOOK_DIR/$hook $HOOK_DIR/$hook-local
+            if [ $make_updates == 1 ]; then
```

Comment:
I believe you are correct that bash is treating them as strings.  To compare them as numbers it would need the `-eq` operator or to use the arithmetic evaluation context `(( ... ))`.  But, the variables can only contain the literal values assigned by the script, so there's nothing unsafe about it unless someone edits the script and breaks it.  If we have an actual standard then I'll update to match.

What's the rational for preferring the `[[` compound command over the `[` or `test` builtin?  (I just matched the style already in use by the other tests in the file, which I suspect are actually buggy and will fail on files with spaces in the filename.  But that's not an issue when comparing the known values 0 and 1.)